### DISCUSSION TITLE
Get final amounts from logs

### DISCUSSION THREAD

**Discussion by tnet-ag** [Elnaril/uniswap-universal-router-decoder#46] [Ideas]
Would like to discuss with you guys what a good workflow / procedure is for enrich the data from this library with finalized amounts from logs.

I do not think that I am the only person who is interested in that.

My tries so far:
Tried to get the finalized amounts from the logs which comes with the recipes (fetched specifically for a tx)

BUT there is somthing missing
specially for multitrade TX
Etherscan is able to parse them, so the data has to be somewhere.

My indexer is currently indexing ALL logs since 17e9 (Block by block)
Will be my next try to look if they are additional logs which are not in the recipes
Can tell soon if this will bring the missing data.

Open for Ideas/feedback :)

**Comment by Elnaril** [comment]
> BUT there is somthing missing
specially for multitrade TX
Etherscan is able to parse them, so the data has to be somewhere.

Could you elaborate on this? Maybe share the corresponding code and transaction? What amount is missing exactly?
